packages/ketcher-core/src/application/editor/shared/coordinates.ts

```diff
@@ -6,7 +6,7 @@ import ZoomTool from '../tools/Zoom';
  * `canvas` -- The real coordinates used to draw entities
  * `view` -- The zoomed canvas coordinates
  */
-class Coordinates {
+export class Coordinates {
```

Comment:
Why have you decided to change it to named export?

Reply:
Because I encountered error when I tried to import the it and I am not sure why. So i changed it.
<img width="571" alt="image" src="https://github.com/epam/ketcher/assets/35161544/bd4cd149-0457-401b-b073-fb2def41f320">

Reply:
you can try to run `npm run build` from the root of the project

Reply:
Same error even run npm run build from the root of the project